testcontainers/src/main/java/org/georchestra/testcontainers/rabbitmq/RabbitmqContainer.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2021 by the geOrchestra PSC
+ *
+ * This file is part of geOrchestra.
+ *
+ * geOrchestra is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * geOrchestra is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * geOrchestra.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.georchestra.testcontainers.rabbitmq;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.Base58;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * A <a href="https://www.testcontainers.org">Testconainers</a> container to run
+ * georchestra's Rabbitmq server as a JUnit {@code @Rule} based on
+ * {@code bitnami/rabbitmq:3.12}.
+ * <p>
+ * Get the host mapped port for {@code 5672} with {@link #getMappedPort(int)
+ * getMappedPort(389)}, or directly with {@link #getMappedRabbitmqPort()}
+ * <p>
+ * For convenience, once the container is started, the mapped Rabbitmq port will
+ * automatically set as a {@link System#getProperty System property}
+ * {@code rabbitmqPort=<mapped port>}, following standard georchestra
+ * datadirectory's property name.
+ */
+public class RabbitmqContainer extends GenericContainer<RabbitmqContainer> {
```

Comment:
I wonder if we could not make use of "org.testcontainers:rabbitmq:1.19.3" (or whatever needed version) as test scope instead ? Is it worth having our own implementation ?